.generator/templates/model/ObjectSerializer.mustache

```diff
@@ -290,3 +294,11 @@ export class UnparsedObject {
     }
 }
 
+export type AttributeTypeMapType = {
```

Comment:
do we want the Type suffix?

Reply:
Not sure, I thought it was a bit more clear.

Reply:
Done.